base/dcp_sharded.go

```diff
@@ -451,6 +451,7 @@ func (c *CbgtContext) Stop() {
 
 func (c *CbgtContext) RemoveFeedCredentials(dbName string) {
 	removeCbgtCredentials(dbName)
+	removeCbgtRootCertsForBucket(c.sourceUUID)
```

Comment:
If multiple databases are using the same bucket, this looks like it's going to delete the shared credentials for all when the  import listener is stopped for any one of the databases using that bucket.